runtime_v10/src/runtime/policy.rs

```diff
@@ -396,3 +393,74 @@ pub mod policy_constants {
     pub const CALIBNET_MINIMUM_CONSENSUS_POWER: i64 = 32 << 30;
     pub const MAINNET_MINIMUM_CONSENSUS_POWER: i64 = 10 << 40;
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use anyhow::*;
+    use quickcheck_macros::quickcheck;
+
+    impl quickcheck::Arbitrary for Policy {
```

Comment:
I would love for this implementation to be public.

Reply:
Put it behind `arb` feature